src/main/java/com/guysfromusa/carsgame/v1/converters/CarConverter.java

```diff
@@ -22,6 +24,8 @@ public static Car toCar(CarEntity carEntity){
         car.setType(carEntity.getCarType());
         car.setDirection(carEntity.getDirection());
 
+        Optional.ofNullable(carEntity.getGame()).map(GameEntity::getId).ifPresent(car::setGameId);
```

Comment:
I think we should not publish internal keys  if we dont need to, API always uses names not ids